site/content/3.12/develop/http-api/schemas.md

```diff
@@ -0,0 +1,1116 @@
+    - and example documents or edges for reference
+
+---
+
+## Supported API Paths
+1. [**GET /_api/schema**](#get-apischema) – Returns all graphs, views, and collections in the database.
+2. [**GET /_api/schema/graph/&lt;graph-name&gt;**](#get-apischemagraphgraph-name) – Returns the specified graph and its connected collections.
+3. [**GET /_api/schema/view/&lt;view-name&gt;**](#get-apischemaviewview-name) – Returns the specified view and its linked collections.
+4. [**GET /_api/schema/collection/&lt;collection-name&gt;**](#get-apischemacollectioncollection-name) – Returns the specified collection and its schema.
+
+---
+
+<h2 id="get-apischema">GET /_api/schema</h2>
+
+```openapi
+paths:
+   /_db/{database-name}/_api/schema:
+      get:
+         operationId: getAllSchemas
+         description: |
+            Show all the information on the database including graphs, views and collections.
+         parameters:
+            - name: database-name
+              in: path
+              required: true
+              example: _system
+              description: |
+                The name of a database that you want to examine its structures and schemas.
+              schema:
+                type: string
+                
+            - name: sampleNum
+              in: query
+              required: false
+              description: |
+                The number of documents/edges to examine per collection when determining
+                attribute types and whether an attribute is optional.
+                Must be a positive integer. Defaults to `100`.
+                If larger than the number of documents, it will only use the available ones.
+              schema:
+                type: integer
+                minimum: 1
+                default: 100
+            
+            - name: exampleNum
+              in: query
+              required: false
+              description: |
+                The number of example documents/edges to return per collection.
+                Must be a non-negative integer and not be larger than `sampleNum`.
+                If `0`, no examples will be returned. Defaults to `1`.
+              schema:
+                type: integer
+                minimum: 0
+                default: 1
+         
+         responses:
+            '200 OK':
+               description: |
+                  The schema overview was successfully returned.
+            '400 Bad Request':
+               description: |
+                  Invalid query parameters (e.g., negative or non-numeric values).
+            '404 Not Found':
+               description: |
+                  Collection, view or graph not found on the database.
+            '405 Method Not Allowed':
+               description: |
+                  When request method is not GET. This endpoint only supports GET.
+            '500 Internal Server Error':
+               description: |
+                  Internal server error.
```

Comment:
It must only be the status code, e.g. `'200':`. We don't tend to describe all of the possible status codes, especially because there can be many different ones if it's not handled by the specific rest handler but somewhere else (e.g. there is always a possibility that a request returns 503 during startup, but describing this for each endpoint would be extremely cumbersome). Which of the listed ones are specific to the schema feature?